alibi/datasets/tensorflow.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple, Union
```

Comment:
@mauicv for these straight swaps I'm taking your word for it that the code is indeed a direct match with the original!